lib/src/builder.dart

```diff
@@ -130,6 +149,13 @@ class PostgrestQueryBuilder extends PostgrestBuilder {
       return c;
     }).join('');
 
+    if (count != null) {
+      headers['Prefer'] = 'count=${count.toString().split('.')[1]}';
```

Comment:
on `count_option.dart` file, you can create extension for this enum to get the string value
```
extension CountOptionName on CountOption {
  String name() {
    return toString().split('.').last;
  }
}

// then you can 
headers['Prefer'] = 'count=${count.name()}';
```

Reply:
Thanks for your tip. This makes the code a lot cleaner!